Prevents supermatter/singularity from pulling anchored items

atlantiscze opened the pull request:
- Delaminating SM/Singulo no longer pulls anchored items (obj/item/), such as, intercoms.

PsiOmegaDelta commented:
Could place the check before the ```spawn(0)``` for some optimization (spawns do come with some cost), or use ```set waitfor = 0``` and sleep() instead.

GinjaNinja32 commented:
`set waitfor = 0` is better in this case.

PsiOmegaDelta commented:
Perhaps even better would be if the pull loop handled doing these things in the background (as much as things run in the 'background' in BYOND).

atlantiscze commented:
Changed to set waitfor = 0 and sleep(), moved anchored check up a bit. Larger changes are out of the scope of this PR, it's suposed to be a fix for small issue.
